gcs-proxy.go

```diff
@@ -173,3 +174,20 @@ out:
 	// recalculate content length
 	f.Response.ReplaceToDecodedBody()
 }
+
```

Comment:
Can you change the message to say:

Maximum object size reached. 
Stream processing in proxy disabled. 
StreamLargeBodies is: xxx MB